### PR TITLE
Update OpenTelemetry CLOMonitor data

### PR DESCRIPTION
**Pull request by jaronoff97**
updates the otel CLO Monitor data

### CODE REVIEW

**Comment by jaronoff97** [comment]
cc @austinlparker @svrnm @trask

**Review by austinlparker — APPROVED** [review]
Thanks for putting this together!

File: data/cncf.yaml [hunk 1]
```diff
@@ -1889,43 +1889,225 @@
...
   maturity: incubating
   repositories:
+    - name: community
+      url: https://github.com/open-telemetry/community
+      check_sets:
+        - community
+    - name: cpp-build-tools
+      url: https://github.com/open-telemetry/cpp-build-tools
+      check_sets:
+        - code
```
Comment (trask) [inline data/cncf.yaml:1908]: (since other build tools repos are excluded also)
```suggestion
```

File: data/cncf.yaml [hunk 2]
```diff
@@ -1889,43 +1889,225 @@
...
+    - name: opentelemetry-rust-contrib
+      url: https://github.com/open-telemetry/opentelemetry-rust-contrib
+      check_sets:
+        - code-lite
+    - name: opentelemetry-sandbox-web-js
+      url: https://github.com/open-telemetry/opentelemetry-sandbox-web-js
+      check_sets:
+        - code
```
Comment (trask) [inline data/cncf.yaml:2072]: my understanding is this is just a playground, and will eventually land in opentelemetry-js
```suggestion
```

**Review by trask — APPROVED** [review]

**Review by tegioz — APPROVED** [review]
Thanks @jaronoff97!

**Pull request merged**